plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFucntionConfigProvider.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.fission;
+
+import java.io.IOException;
+
+public class FissionFucntionConfigProvider
```

Comment:
```suggestion
public class FissionFunctionConfigProvider
```
Don't forget to change the file name as well!

Reply:
Javadoc?

Reply:
Trino doesn't do Javadoc. It seems like a deliberate choice to make the code less hefty. Honestly, I think I like it.